core/src/main/scala/com/mesosphere/usi/core/logic/MesosEventsLogic.scala

```diff
@@ -119,6 +121,7 @@ private[core] class MesosEventsLogic(mesosCallFactory: MesosCalls) extends Impli
       case UpdateEvent(taskStatus) =>
         val taskId = TaskId(taskStatus.getTaskId.getValue)
         val podId = podIdFor(taskId)
+        logger.info(s"Received task status update from taskId $taskId and podId $podId with status ${taskStatus.getState}", LoggingArgs(("taskId", taskId), ("podId", podId)))
```

Comment:
since we are using LoggerTakingImplicit, the LoggingArgs needs to be curried to `info` method [like this](https://github.com/mesosphere/usi/blob/9cb81b020c323faf20e951284db536ee805a815d/core/src/main/scala/com/mesosphere/package.scala#L56):
```
logger.info(s"Received task status update from taskId $taskId and podId $podId with status ${taskStatus.getState}")(LoggingArgs("taskId" -> taskId).and("podId"->podId))
```

Reply:
oh right 🙈 oh wow, that's error prone!